src/components/Icons/Svg/SvgIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export interface SvgIconProps {
+  svg: React.ReactNode;
+  bg: string;
+}
+
+export function SvgIcon({ ...props }) {
+  return (
+    <div
+      className={`mr-2 bg-primary h-6 w-6 rounded-full`}
```

Comment:
Trying to understand why do we really need a margin right!? 🤔

Reply:
will change !